contributing/start.rst

```diff
@@ -56,5 +99,135 @@ Thanks! ❤ ❤ ❤
 
   Zammad Team
 
+.. _translation:
+
+Translation
+============
+
+If you want to contribute to the translations and improve the multi-language
+support of Zammad and/or the documentation, you are welcome to contribute
+as well! The translation of Zammad itself and the documentation is done by using
```

Comment:
`If you want to contribute to the translations` => `If you want to help us with translation`